cms/templates/partials/card_details_bottom.html

```diff
@@ -0,0 +1,39 @@
+{% load static humanize wagtailcore_tags wagtailimages_tags %}
+
+<div class="bottom">
+  {% with page_id=courseware_page.pk|stringformat:"i" %}
+  {% with collapse_id="collapsible_"|add:card_type|add:"_"|add:page_id collapse_input_id="collapse_"|add:card_type|add:"_"|add:page_id %}
```

Comment:
You need to `|add:tab` as well, otherwise say, a program is featured, expanding the details on the featured card also expands the card in "All" tab as well as the "Programs" tab.

Reply:
`card_type` value would `featured` or `tab` according to product.

Reply:
@tasawernawaz I meant value of the `tab` variable. It cycles between `all`, `program` and `course` values. You need to include this string in the `collapse_id` and `collapse_input_id` in order for cards to not affect other cards whenever details are expanded/collapsed.